readme.md

```diff
@@ -35,6 +35,15 @@ NOTE: It is strongly recommended that you use python virtual environments.  Virt
 
 ## Release Version History
 
+### Version 0.10.4
+
+* Features:
+  * If multiple classes are found (ie when searching for a SettingsManager), it will now pick the one that is deepest in hierarchy from the desired class
+  * Updated parsers for DSC, FDF, and DEC
+* Bugs
+  * Fixed bugs #42, 48, 49, and 51
```

Comment:
i didn't see 42 in the milestone

Reply:
It wasn't in the milestone but we've merged it since the last release so I figured why not add it

Reply:
can we merge in the #53 so that all the class resolution bugs are in